### PR TITLE
Fix contact send message button omnibox

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by rowanseymour** [comment]
Yup

**Comment by nicpottier** [comment]
looks good, @ericnewcomer ?

File: templates/contacts/contact_read.haml [hunk 1]
```diff
@@ -642,7 +642,7 @@
     $(".contact-send-button").on('click', function() {
       {% if has_sendable_urn %}
       $(".gear-menu").removeClass("open");
-      showComposeInitialized("c={{object.id}}", null, fetchRecentMessages);
+      showComposeInitialized("c={{object.uuid}}", null, fetchRecentMessages);
```
Comment (ericnewcomer) [inline templates/contacts/contact_read.haml]: doh.

**Comment by ericnewcomer** [comment]
Yup. Sorry for the uuid turds. Looks good.

**Pull request merged**